README.md

```diff
@@ -3,3 +3,12 @@
 Simple ansible role for installing and configuring influxdb.
 
 Designed to be used in OpenWISP installations.
+
+Role Variables
+==============
+
+```yaml
+# By default HTTP query logging of InfluxDB is turned off since it can occupy a
+# lot of space. You can turn it on by setting below variable to true
+ansible_ow_influxdb_http_logging: true
```

Comment:
I would call this simply `influxdb_query_logging` and would refer to it as query logging instead of HTTP query logging since we're managing both HTTP logging and query logging, what do you think?